### ISSUE TITLE
v2.3.0 Whoops\Exception\ErrorException thrown with message "Only variables should be passed by reference"

### ISSUE THREAD

**Issue by stsimb** [paulmassen/grav-plugin-seo#58]
Hi,

I've updated from v2.2.0 to v2.3.0 and now getting this error message

```
Whoops\Exception\ErrorException thrown with message "Only variables should be passed by reference"

Stacktrace:
#13 Whoops\Exception\ErrorException in /var/lib/nginx/test-gr/home/user/plugins/seo/seo.php:240
#12 Whoops\Run:handleError in /var/lib/nginx/test-gr/home/user/plugins/seo/seo.php:240
#11 Grav\Plugin\seoPlugin:onPageInitialized in /var/lib/nginx/test-gr/home/vendor/symfony/event-dispatcher/EventDispatcher.php:184
#10 call_user_func in /var/lib/nginx/test-gr/home/vendor/symfony/event-dispatcher/EventDispatcher.php:184
#9 Symfony\Component\EventDispatcher\EventDispatcher:doDispatch in /var/lib/nginx/test-gr/home/vendor/symfony/event-dispatcher/EventDispatcher.php:46
#8 Symfony\Component\EventDispatcher\EventDispatcher:dispatch in /var/lib/nginx/test-gr/home/vendor/rockettheme/toolbox/Event/src/EventDispatcher.php:23
#7 RocketTheme\Toolbox\Event\EventDispatcher:dispatch in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Grav.php:283
#6 Grav\Common\Grav:fireEvent in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Processors/PagesProcessor.php:26
#5 Grav\Common\Processors\PagesProcessor:process in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Grav.php:132
#4 Grav\Common\Grav:Grav\Common\{closure} in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Grav.php:374
#3 Grav\Common\Grav:Grav\Common\{closure} in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Grav.php:350
#2 call_user_func_array in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Grav.php:350
#1 Grav\Common\Grav:__call in /var/lib/nginx/test-gr/home/system/src/Grav/Common/Grav.php:133
#0 Grav\Common\Grav:process in /var/lib/nginx/test-gr/home/index.php:52
```

Reverted to v2.2.0 until it's fixed.

**Comment by stsimb** [comment]
Line 240 was committed about an hour ago with commit https://github.com/paulmassen/grav-plugin-seo/commit/a0979ce073965e445e9fb87c6ba84666221e35bd

**Comment by paulmassen** [comment]
Thanks, I forgot to remove that, can you try this release? https://github.com/paulmassen/grav-plugin-seo/releases/tag/v2.3.1

**Comment by stsimb** [comment]
Yeah, commenting out line 240 fixes the problem!

👍

**Comment by paulmassen** [comment]
Fixed with 2.3.4 release
